redCOMPONENT-COM/redSHOP
# REDSHOP-1379 #comment API has been changed.

testing details

1) enable packsoft integration from configuration 
2) provide username/pass (I dont have testing details, I used user's details)
3) go to order listing, you will see create label button,
4) click on it, you should be able to create label successfully
